File: sdk/javascript/examples/basic.js
import { EvaluationContext, FeatureFlagClient } from "../src/index.js";

const client = new FeatureFlagClient({
  baseUrl: "http://localhost",
  projectKey: "checkout",
  environmentKey: "production",
});

const context = new EvaluationContext("user-107", {
  premium: true,
  country: "US",
  plan: "pro",
});

const enabled = await client.isEnabled("new_checkout", context);

console.log({ enabled });

File: sdk/python/examples/basic.py
from featureflags import ClientConfig, EvaluationContext, FeatureFlagClient

client = FeatureFlagClient(
    ClientConfig(
        base_url="http://localhost",
        project_key="checkout",
        environment_key="production",
    )
)

context = EvaluationContext(
    user_id="user-107",
    attributes={
        "premium": True,
        "country": "US",
        "plan": "pro",
    },
)

if client.is_enabled("new_checkout", context):
    print("Render new checkout")
else:
    print("Render legacy checkout")
